docs/ctapipe_api/io/index.rst

```diff
@@ -131,10 +131,48 @@ information. It can be used in an event loop like:
         for event in source:
             calibrate(event)
             write_data(event)
-    
+
 Reading Output Tables:
 ======================
-In addition to using an `EventSource` to read R0-DL1 data files, one can also access full *tables* for files that are in HDF5 format (e.g. DL1 files).
+In addition to using an `EventSource` to read R0-DL1 data files, one can also access full *tables* for files that are in HDF5 format (e.g. DL1 and higher files).
+
+
+A convenient way to load tables that directly contain the subarray- or telescope-wise
+event data for multiple telescopes and different primitive tables in the files is the
+`~ctapipe.io.TableLoader`:
+
+.. code-block:: python
+
+    from ctapipe.io import TableLoader
+
+    loader = TableLoader("events.dl1.h5")
+    events = loader.read_subarray_events()
+    tel_events = loader.read_subarray_events()
+
+    print(loader.subarray, len(events), len(tel_events))
+
+
+Loading the DL1 image data for telescopes with different numbers of pixels
+does not work as astropy tables do not support heterogenous data in columns.
+In this case, use:
+
+.. code-block:: python
+
+    from ctapipe.io import TableLoader
+
+    loader = TableLoader("events.dl1.h5", load_dl1_images=True)
+
+    # tel_events is now a dict[str] -> Table per telescope type
```

Comment:
I think this is not really clear.
It is one dict with telescope types as keys and the tables as values and not _a dict (...) per telescope type_, right?

Reply:
(Table per telescope type)

Reply:
Maybe use type-hint syntax:

 `dict[str, Table]`  which maps a telescope type string to the table of events for that telescope